index.html

```diff
@@ -640,6 +641,16 @@ <h2>Typography</h2>
 				<h2>Spinner</h2>
 				<ts-spinner data-visible data-size="medium" data-color="mono"> </ts-spinner>
 			</article>
+			<article>
+				<h2>Form</h2>
+				<h3>Radio</h3>
+				<ts-radio name="radio" value="red" data-label="Red pill" checked disabled dir="rtl"></ts-radio>
```

Comment:
what about `ts-radio-group`? It seems to be similar to the `tabs` component, so maybe we can have similar approach for theme?